app/src/crypto.c

```diff
@@ -575,15 +575,15 @@ uint32_t crc32_for_byte(uint8_t rbyte) {
     return r ^ (uint32_t) 0xFF000000L;
 }
 
-void crc32_small(const void *data, uint8_t n_bytes, uint32_t *crc) {
+void crc32_small(const void *data, uint16_t n_bytes, uint32_t *crc) {
     for (uint8_t i = 0; i < n_bytes; ++i) {
```

Comment:
index from loop for should be at least uint16_t

Reply:
good catch, solved :)